templates/cortex-values.yaml

```diff
@@ -0,0 +1,55 @@
+---
+config:
```

Comment:
are all these values non default-one ? if they are not custom configuration, we should let them empty. Also to allow terraform interpolation if needed they sould go in `local.values_cortex` (I know it bloats the code a little but all the other addons are made like this)